tests/xcm-simulator/build.rs

```diff
@@ -131,7 +131,7 @@ fn main() {
     fs::create_dir_all(&config.fixtures_dir).unwrap();
 
     // build all the contracts
-    build_contracts(&config, ["flipper"].to_vec());
+    build_contracts(&config, ["flipper", "async-xcm-call-no-ce"].to_vec());
```

Comment:
Some config file might be a better option than changing the `build.rs` each time.

Reply:
Or better, no need for either. It will simply pick all the contracts from `contracts_dir`. Much simple solution.
Added in https://github.com/AstarNetwork/Astar/pull/941/commits/8b6f9b1b3d82bf599b8ef7836726463a2bd18ff5